code/_helpers/global_lists.dm

```diff
@@ -44,6 +44,11 @@ var/global/list/poster_designs = list()
 // Uplinks
 var/list/obj/item/device/uplink/world_uplinks = list()
 
+
+//Neotheology
+var/global/list/all_rituals = list() //List of all rituals
+var/list/global_ritual_cooldowns = list() // internal lists. Use ritual's cooldown_category
```

Comment:
Put this inside `/datum/ritual/` define as `var/global/list/global_ritual_cooldowns = list()`

Reply:
I just took it FROM there
Its a global list, global_lists.dm is the best place for it. it's the defacto location for global lists

Reply:
If you want it to be a global accessible to everything, why are you not using `GLOB.`?

Reply:
I actually don't know how to use glob, i wanted to

will look into it, ive also asked cups for help since he ported it